### PR TITLE
update lambda to use cfnresponse

### PR DESCRIPTION
**Pull request by steved**
When building a custom response, we were returning the resource property data which could exceed the 4k response limit.

### CODE REVIEW

File: cdk/domino_cdk/provisioners/lambda_files/backup_post_creation_tasks.py [hunk 1]
```diff
@@ -40,4 +33,3 @@ def on_delete(event):
         response = client.delete_recovery_point(
```
Comment (fraenkel) [inline cdk/domino_cdk/provisioners/lambda_files/backup_post_creation_tasks.py:33]: we shouldn't set response

**Review by fraenkel — APPROVED** [review]
hmm, we dont have tests?

File: cdk/domino_cdk/provisioners/lambda_files/backup_post_creation_tasks.py (hunk 1 above)
Comment (steved) [reply cdk/domino_cdk/provisioners/lambda_files/backup_post_creation_tasks.py:33]: yep, good catch

**Comment by steved** [comment]
Will follow-up here for unit tests: https://github.com/dominodatalab/cdk-cf-eks/pull/119

**Pull request merged**